lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js

```diff
@@ -27,12 +27,13 @@ const Gatherer = require('../gatherer');
 function collectTagsThatBlockFirstPaint() {
   return new Promise((resolve, reject) => {
     try {
-      const tagList = [...document.querySelectorAll('link, head script[src]')]
+      const tagList = [...document.querySelectorAll('link, head script[src], script[type]')]
         .filter(tag => {
           if (tag.tagName === 'SCRIPT') {
             return !tag.hasAttribute('async') &&
                 !tag.hasAttribute('defer') &&
-                !/^data:/.test(tag.src);
+                !/^data:/.test(tag.src) &&
+                !(tag.getAttribute('type') === 'module');
```

Comment:
let's do `tag.getAttribute('type') !== 'module'`

Reply:
is this just for readability or anything I'm missing over?
I just wrote like this to make it consistent with the above checks

Reply:
well the others are like that only because there not using an operator, so yeah just easier to read :)